src/config/networks/network.d.ts

```diff
@@ -22,7 +22,8 @@ export enum FEATURES {
   ERC721 = 'ERC721',
   ERC1155 = 'ERC1155',
   SAFE_APPS = 'SAFE_APPS',
-  CONTRACT_INTERACTION = 'CONTRACT_INTERACTION'
+  CONTRACT_INTERACTION = 'CONTRACT_INTERACTION',
+  ENS = 'ENS',
```

Comment:
How about setting a more semantic name? For example `ENS_RESOLVE` ?

Reply:
replaced with `ENS_LOOKUP` as decided internally. [`ae7e770` (#1570)](https://github.com/gnosis/safe-react/pull/1570/commits/ae7e7705757c648f3283428d328dcae13549b5be)